src/bin/pgcopydb/parsing_utils.c

```diff
@@ -782,7 +782,7 @@ buildPostgresURIfromPieces(URIParams *uriParams, char **pguri)
 		char *keyword = uriParams->parameters.keywords[index];
 		char *value = uriParams->parameters.values[index];
 
-		if (value != NULL && !streq(value, ""))
+		if (!streq(value, ""))
```

Comment:
Same reasoning with later refactoring opportunities and leaving clues around.

Reply:
The reason for the change is that `streq` already has the null checks by definition.